src/components/product/ProductCard.tsx

```diff
@@ -23,7 +23,7 @@ export const ProductCard: React.FC<ProductCardProps> = ({
     return (
         <div
             data-testid={`product-card-${id}`}
-            className='aspect-card w-card-width rounded-lg bg-grey-100 outline outline-1 outline-grey-200'
+            className='rounded-lg bg-grey-100 outline outline-1 outline-grey-200 sm:w-full'
```

Comment:
The cards width should be fixed. 

<img width="1429" alt="Screenshot 2024-10-10 at 21 52 57" src="https://github.com/user-attachments/assets/e99ac96a-2640-4395-a251-562f81152047">
h